src/main/java/com/redhat/cloud/policies/app/EngineResponseExceptionMapper.java

```diff
@@ -55,8 +54,9 @@ public RuntimeException toThrowable(Response response) {
     private Msg getBody(Response response) {
         String msg = response.readEntity(String.class);
         if (msg != null && !msg.isEmpty()) {
-            try (Jsonb jsonb = JsonbBuilder.create()) {
-                Map<String, String> errorMap = jsonb.fromJson(msg, HashMap.class);
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
This `ObjectMapper` could be injected. Unsure if this class accepts `@Inject`s but maybe it will do as I see it has a `@Priority` annotation. didn't test.